src/NHibernate/Hql/Ast/ANTLR/ASTQueryTranslatorFactory.cs

```diff
@@ -14,11 +14,6 @@ namespace NHibernate.Hql.Ast.ANTLR
 	/// </summary>
 	public class ASTQueryTranslatorFactory : IQueryTranslatorFactory
 	{
-		public IQueryTranslator[] CreateQueryTranslators(string queryString, string collectionRole, bool shallow, IDictionary<string, IFilter> filters, ISessionFactoryImplementor factory)
```

Comment:
This one was obsolete in `IQueryTranslatorFactory` but not in its concrete implementation. If we want to be conservative, then we should flag it obsolete instead of removing it.

Reply:
That's fine.